skynet.yaml

```diff
@@ -0,0 +1,16 @@
+# Testing with Skynet - https://github.com/Workiva/skynet/tree/master/docs
+name: unit-tests
+image: golang:1.18-bullseye
+description: run unit tests
+size: small
+timeout: 600 # 10 minutes
+contact: support-onecloud
+
+run:
+  on-pull-request: true
+  on-tag: true
+  when-branch-name-is:
+    - master
+
+scripts:
+  - echo This repository does not run any tests
```

Comment:
can we add the tests and vet here?

Reply:
The tests are broken; they don't work on the master branch before these changes were applied. That's why I'm not running them.

As for vet - we're not currently running vet in any of these repos. The primary purpose of the migration was to move the repos over with the least amount of modification possible. Vet wasn't being run before, and so I haven't been adding it in. However, this is something we could add in the future.

Reply:
got it, ok